lightning_examples/mnist-hello-world/hello-world.py

```diff
@@ -97,8 +201,70 @@ def configure_optimizers(self):
 
 
 # %%
+
+
 class LitMNIST(L.LightningModule):
-    def __init__(self, data_dir=PATH_DATASETS, hidden_size=64, learning_rate=2e-4):
+    """PyTorch Lightning module for training a multi-layer perceptron (MLP) on the MNIST dataset.
+
+    Attributes
+    ----------
```

Comment:
lets unify the documentation style and mostly we use Napoleon

Reply:
I have changed the documentation style to Napoleon Google Format